src/01-functions.js

```diff
@@ -24,8 +26,12 @@ function alwaysTrue() {}
  *  greet("Ian");
  *  //> "Hello, Ian."
  */
-function greet() {}
+function greet(name) {
+  name.charAt(0).toUpperCase;
```

Comment:
I like this extra step here to ensure the name is capitalized.  Can we do this without ` charAt() `